src/document.cpp

```diff
@@ -355,6 +355,20 @@ rvoe<NoReturnValue> PdfDocument::add_page(std::string resource_dict,
     return NoReturnValue{};
 }
 
+rvoe<NoReturnValue> add_page_label(PageId page_id, std::string label) {
+    page_labels.append({page_id, CAPY_PAGE_LABEL_NUMBER_STYLE_NONE, label, 0});
+    return NoReturnValue{};
+}
+
+rvoe<NoReturnValue> add_page_labeling(PageId page_id,
+                                      CapyPDF_PageLabelNumberStyle style,
+                                      std::string prefix = "",
```

Comment:
This should be a u8string and it should either be taken as a const reference or be `std::move` into the array below.